### PR TITLE
outline repo in readme

### PR DESCRIPTION
**Pull request by MattHJensen**
Proposed outline for this repo. 

If we adopt this outline, I imagine we would will label issues by project using an "ex#" tag. So anything that has to do with the first paper would use an ex1 label and so forth. 

cc @econ02 @andersonfrailey @Amy-Xu

### CODE REVIEW

**Comment by MattHJensen** [comment]
@Amy-Xu, could you review this PR and merge if it looks good? 

@andersonfrailey, after this is merged, could you open a PR adding the ex1 materials as indicated in the readme?

**Comment by Amy-Xu** [comment]
- Do we need to add a brief summary for each example?
- I imagine most users of this repo do not need the link to google doc, do they? An alternative is to contact you for the editable version. A side issue that I have been thinking about for a while is that should we move all those google docs to shared google account?
- Instead of having a empty example 2, may we could have a section of prospective ideas that includes more specifics like transitional solution etc?

**Comment by MattHJensen** [comment]
@amy-xu asked:

> Do we need to add a brief summary for each example?

See the latest commit that adds abstracts in their own section underneath (this follows the NBER working paper newsletter's style)

> I imagine most users of this repo do not need the link to google doc, do they?

The reason I included that is to make it as easy as possible for people to contribute to the project. What if we include a link that allows for commenting but not editing:

image

**Comment by MattHJensen** [comment]
@Amy-Xu said: 

> A side issue that I have been thinking about for a while is that should we move all those google docs to shared google account?

Who would own the google doc? I don't think they should be  "OSPC" owned as they don't have anything to do with the webapps. I'm inclined to think they should be owned by one of the authors -- perhaps it should be me. 

> Instead of having a empty example 2, may we could have a section of prospective ideas that includes more specifics like transitional solution etc?

I agree, as soon as we have an working title for the second section we should update from "untitled" to the working title. But I didn't want to have to resolve the working title for ex2 in this PR, so I left it as is for now.

**Comment by Amy-Xu** [comment]
@MattHJensen 

>The reason I included that is to make it as easy as possible for people to contribute to the project. What if we include a link that allows for commenting but not editing:

That sounds good to me.

LGTM. Merging.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+- [Link to "Working Paper PDF" in repo at ex1/UBI-working-paper.pdf]
+- [Link to "Working Paper Google Doc" that generates the PDF.]
+- [Link to "Scripts" that generates tables for working paper in repo at ex1/filename.pdf] 
+
+### Ex. 2: Untitled, Under Development 
+- [Link to Development Folder in repo at ex2/]
+
+## Abstracts
+
+### Ex. 1: A Budget-Neutral Universal Basic Income, AEI Economics Working Paper 2017-10
```
Comment (Amy-Xu) [inline README.md:16]: hmmmmm I just merged the PR, but somehow missed this: why is it October?

Comment (MattHJensen) [reply README.md:16]: That's AEI's number for the WP. It is the 10th working paper in 2017

**Pull request merged**